web/src/pages/_app.tsx

```diff
@@ -31,7 +31,7 @@ function Account({ Component, pageProps: { ...pageProps } }: AppProps) {
         <BottomNav />
         <Toaster
           position={"bottom-center"}
-          containerClassName={"!bottom-16 !select-none"}
+          containerClassName={"!bottom-16 !select-none !text-center"}
```

Comment:
If you're not aware, tailwind provides [a way to always use `!important`](https://tailwindcss.com/docs/configuration#important) in case you ever need it. I think there's no need for now, but something to consider in the future if we need a lot of `!important` styles.

Reply:
I'm just doing this here since I only need it for this toaster component. Using !important tends to not be good practice but in this case I'm forced to since im overriding existing styling

Reply:
So I generally agree, but using that option can be useful for working around obscure CSS specificity issues. I've had cases where the odd id selector or something overrides tailwind, and not having to worry about that can be nice. 
https://sebastiandedeyne.com/why-we-use-important-with-tailwind/ is interesting reading. No issue here though